src/js/publish-wasm-dependecy.js

```diff
@@ -56,6 +57,39 @@ const wasmPackageJson = `{
 }
 `;
 
+const packageReadme = `# vectorized wasm-api
```

Comment:
why not simply read a file here, instead of escaping it? load it up in the builder 

so we can simply edit `.md` without editing code.